en/docs/services/service-concepts.md

```diff
@@ -1,46 +1,43 @@
+ - **Slack**: Connects your service to Slack API to perform operations in Slack, such as sending a message to a channel, deleting a message, etc.
+ - **GitHub**: Connects your service to GitHub API to perform operations in GitHub, such as creating an issue, creating a pull request, etc. 
+ - **Gmail**: Connects your service to Gmail API to perform email operations such as sending an email, receiving an email, etc.
+ - **Google Calendar**: Connects your service to Google Calendar API to perform operations in Google Calendar such as creating an event, deleting an event, etc.
+ - **Google Sheets**: Connects your service to Google Sheets API to perform operations in Google Sheets such as reading data, formatting data, etc.
+ - **Salesforce**: Connects your service to Salesforce API to perform Salesforce operations such as obtaining organization information, creating a record, etc.
+ - **NetSuite**: Connects your service to Netsuite API to perform Netsuite operations such as creating a record instance, creating a sublist, etc.
 
 ## Statement
-A statement in Choreo services is a syntactic unit that specifies the action to be carried out in a Choreo service. 
+A statement is a collection of code fragments that specifies the actions to be carried out by a Choreo service. Choreo statements can also be composed of expressions. These expressions are written based on the [Ballerina Expression Syntax](#ballerina-expression-syntax).
 
 Choreo supports the following statements:
 
- - **Log**: To add an entry to the service log at runtime. You can either add an **Info** log or an **Error** log depending on your requirement.   
- - **Variable**: To declare a variable of a preferred type depending on your requirement.
- - **If**: To perform a conditional check and evaluate whether a statement is true or false.   
- - **ForEach**: To iterate over a list of items to perform control flow. 
- - **While**: To iterate over a given condition to perform control flow. 
- - **Respond**: To send a response back to the client. For example, to get an API to respond with an `HTTP_OK` indicating that the request was processed successfully.
- - **Custom**: To specify a custom statement depending on your requirement.
+ - **Log**: Logs an event with an information statement or an error that occurs in your service.    
+ - **Variable**: Declares a variable and a value of a preferred data type (string, integer, etc.) to use later in the logical process of the service.
+ - **If**: Specifies two blocks of logical components so that the system can decide which block to execute based on whether the provided condition is true or false.   
+ - **ForEach**: Includes a control flow statement in your service to iterate over a list of items.
+ - **While**: Continuously executes a block of statements in a loop as long as the specified condition is TRUE. 
```

Comment:
Why are we using uppercase TRUE here? Also in the if condition we have used `true or false`. Use consistent wordings.

Reply:
It is based on context. But I guess it could be confusing. Hence, reworded.